taxi/views.py

```diff
@@ -34,6 +35,26 @@
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.edit.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_form.html"
+
+
+class ManufacturerUpdateView(LoginRequiredMixin, generic.edit.UpdateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_form.html"
```

Comment:
The same comment as above applies here. You don't need to override `template_name` if it's the same as the default one. The default `template_name` for an UpdateView is `<app name>/<model name>_form.html`. So in this case, you don't need to specify `template_name` as `taxi/manufacturer_form.html` because it's the default one.